app/src/main/java/com/example/next_show/data/RecommendationClient.java

```diff
@@ -25,13 +26,37 @@
     public static final int FORBIDDEN_REQUEST = 403;
     public static final int LIMIT = 5;
 
+    // instance vars
+    private Shows showsObj;
+    private FeedFragment.GenreMatchedCallback genreMatchedCallback;
+    private FeedFragment.ShowCallback showCallback;
+
     // empty constructor
     public RecommendationClient() { }
 
-    public void fetchRelatedShows(Shows traktShows, List<String> savedShows, ResponseCallback callback) {
+    public RecommendationClient(Shows showsObj) {
+        this.showsObj = showsObj;
+    }
+
+    public void fetchRecommendedShows(List<String> savedShows, FeedFragment.ShowCallback showCallback, FeedFragment.GenreMatchedCallback gmCallback) {
```

Comment:
Here, you should just take one callback of type: `ResponseCallback`.  Then you'll want to pass in the actual feed implementation.  Otherwise, how would another fragment be able to reuse this method call?